### ISSUE TITLE
Aikau Dashlet - Title bar actions defined to be transparent

### ISSUE THREAD

**Issue by stevenokennedy** [Alfresco/Aikau#1151]
The css file for alfresco/dashlets/Dashlet defines the title-bar-actions element, where e.g. config or help icons are normally displayed in dashlets to be completely transparent by setting opacity to 0:

```css
&__title-bar-actions {
      height: 16px;
      **opacity: 0;**
      position: absolute;
      right: 4px;
      top: 1px;
      visibility: visible;
      .titleBarActionIcon {
         background-repeat: no-repeat;
         cursor: pointer;
         float: right;
         height: 16px;
         margin: 3px;
         width: 16px;
      }
```

This is not a sensible default as it means that anyone who wants to place widgets within the widgetsForTitleBarActions container that the Dashlet provides will have to override the css to make these widgets opaque

**Comment by draperd** [comment]
Thanks for raising this issue. I've created https://issues.alfresco.com/jira/browse/AKU-1037 to capture the problem and it will be addressed in the next sprint. I can see that we have code that controls the `display` state when widgets are provided (see the calls to the `processContainer` function) but the CSS class that is applied (`alfresco-dashlets-Dashlet--has-widgets`) does not change the opacity. I'll investigate this and resolve.

**Comment by draperd** [comment]
The fix for this is now available in the 1.0.78 release that is now available.
